### PR TITLE
alert rule history skeleton using static data

### PR DESCRIPTION
**Pull request by ahmadshaheer**
#### Related Issues / PR's
close https://github.com/SigNoz/engineering-pod/issues/1669
close https://github.com/SigNoz/engineering-pod/issues/1613

#### Screenshots
https://www.loom.com/share/4ba6ce623cfc492c9a775782db2a6aa2
NA

### CODE REVIEW

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	}
+	return (
+		<div
+			className={`change-percentage ${
+				direction ? 'change-percentage--success' : 'change-percentage--error'
+			}`}
+		>
+			<div className="change-percentage__icon">
+				{direction ? (
+					<ArrowDownLeft size={14} color="var(--bg-forest-500)" />
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:29]: use Colors

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+				) : (
+					<ArrowUpRight size={14} color="var(--bg-cherry-500)" />
+				)}
+			</div>
+			<div className="change-percentage__label">
+				{percentage}% vs Last {duration}
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:35]: % -> use icon

File: frontend/src/container/AlertHistory/Statistics/TotalTriggeredCard/totalTriggeredCard.styles.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.total-triggered-card {
+	width: 21.7%;
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Statistics/TotalTriggeredCard/totalTriggeredCard.styles.scss:2]: Why specifically 21.7% ?

**Review by YounixM — APPROVED** [review]

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:24]: Done

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx (hunk 2 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx]: @YounixM , in the designs, we have text % (https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7734&m=dev)

File: frontend/src/container/AlertHistory/Statistics/TotalTriggeredCard/totalTriggeredCard.styles.scss (hunk 3 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Statistics/TotalTriggeredCard/totalTriggeredCard.styles.scss]: I had calculated it. 
total width of the stats container= 1348 pixels
stats card width = 293 pixels. 

`293 * 100 / 1348 = 21.73% `

https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7717&m=dev

**Pull request merged**